### ISSUE TITLE
resolve() don't work for symlinks created by mklink on windows

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vim/vim#147]
```
What steps will reproduce the problem?
1. in cmd create symlink: mklink "c:/target" "c:/source"
2. in vim: echo resolve("c:/target");

What is the expected output? What do you see instead?
I expect "c:/source" in output, because "c:/target" is pointed to "c:/source", 
but get "c:/target" instead.

What version of the product are you using? On what operating system?
Latest vim 7.3.1280
Windows8
```

Original issue reported on code.google.com by `z...@softvisio.net` on 3 Jul 2013 at 8:19

**Comment by wsdjeg** [comment]
Any update with this issue? still get this issue in window7 when using vim 8.1

**Comment by k-takata** [comment]
Fixed by 8.1.0894: https://github.com/vim/vim/commit/dce1e89be4675bcdbc9785584d3da25295481e63

**Comment by wsdjeg** [comment]
Thanks
